packages/integration-sdk-cli/src/__tests__/cli-run.test.ts

```diff
@@ -80,92 +74,19 @@ test('disables graph object schema validation', async () => {
   expect(process.env.ENABLE_GRAPH_OBJECT_SCHEMA_VALIDATION).toBeUndefined();
 });
 
-test('step should fail if enableSchemaValidation = true', async () => {
```

Comment:
I removed these two tests. It would be somewhat difficult to test them now that the run command doesn't rely on the `log` package. It's just a boolean flag and the actual implementation is tested elsewhere.